src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -1,5 +1,6 @@
 import errno
 import json
+import yaml
```

Comment:
could you add `pyyaml` to `/ceph/src/pybind/mgr/requirements.txt` ?

Reply:
Maybe also `ceph.spec.in`? I hit import error if pyyaml package is not installed.